microlens-pro/microlens-pro.cabal

```diff
@@ -7,8 +7,8 @@ description:
   .
   Thank you to the <https://hackage.haskell.org/package/lens lens> contributors
   for the original code and some docs,
-  <https://cohomolo.gy Emily Pillmore> and
-  <https://mroman42.github.io/cosmoi Mario Román> for
+  Emily Pillmore (<https://cohomolo.gy link>) and
+  Mario Rom&#225;n (<https://mroman42.github.io/cosmoi link>) for
```

Comment:
HTML escaping doesn't seem to work inside links in Hackage, so I pulled out the link